test/utils/daemon-factory.js

```diff
@@ -1,18 +1,19 @@
-
 import { createFactory } from 'ipfsd-ctl'
-import isNode from 'detect-node'
+import { isNode } from 'wherearewe'
 
 export async function daemonFactory () {
   let ipfsHttpModule
   let ipfsModule
 
   try {
+    // @ts-expect-error env var could be undefined
     ipfsHttpModule = await import(process.env.IPFS_JS_HTTP_MODULE)
   } catch {
     ipfsHttpModule = await import('ipfs-http-client')
```

Comment:
Elsewhere you use `await import(process.env.IPFS_JS_HTTP_MODULE || 'ipfs-http-client')` should we create a wrapper for the module import and use that everywhere?